pgjdbc/src/main/java/org/postgresql/util/LruCache.java

```diff
@@ -116,11 +165,8 @@ public LruCache(int maxSizeEntries, long maxSizeBytes, boolean accessOrder,
    * @throws SQLException if entry creation fails
    */
   public synchronized Value borrow(Key key) throws SQLException {
-    Value value = cache.remove(key);
+    @Nullable Value value = cache.remove(key);
```

Comment:
`@Nullable` is not needed for the local variables, as it is inferred automatically.

Reply:
So does that mean the `@Nullable` in `put`[1] is not needed either? I added this here to be consistent with that.
I still have not gotten the hang of when these are needed and what the defaults are.

[1] - https://github.com/pgjdbc/pgjdbc/blob/master/pgjdbc/src/main/java/org/postgresql/util/LruCache.java#L145

Reply:
Yep. Shouldn't be needed. I've never seen it used that way before. :-)

Reply:
@bokken I guess the only thing left here is to remove the @Nullable in both places, then we can merge this.